docs/file-reference.md

```diff
@@ -423,6 +423,41 @@ More info about Probe: https://kubernetes.io/docs/api-reference/v1.6/#probe-v1-c
 
 The name of the Ingress.
 
+## routes
+
+```yaml
+routes:
+- <routeObject>
+- <routeObject>
+```
+
+| **Type**                                  | **Required** |
+|-------------------------------------------|--------------|
+| array of [route object](#routeObject) | no           |
+
+
+### routeObject
+
+```yaml
+routes:
+- name: <string>
+  <Route Spec>
+```
+
+Example:
+```yaml
+name: webroute
+host: httpd-web.192.168.42.69.nip.io
+to:
+  kind: Service
+  name: httpd
+  weight: 100
```

Comment:
I would remove `weight` and `wildcardPolicy` It just makes it look too complicated